spec/book_spec.rb

```diff
@@ -0,0 +1,13 @@
+require 'spec_helper'
+require_relative '../lib/models/book'
+
+describe Book do
+  subject(:book) { Book.new('Sample Title', 'Sample Author') }
+
+  describe '#initialize' do
+    it 'initializes with given title and author' do
+      expect(book.title).to eq('Sample Title')
+      expect(book.author).to eq('Sample Author')
+    end
```

Comment:
- [ ] Great job working on the tests so far team but think it would be great if you add the tests for the classes that you created while implementing the Decorator design pattern like `Decrator` and ` namable` and `trimmer` and `capitalize` classes. 💯🤩🔥


![image](https://github.com/lerfast/opp-school-library/assets/58553711/80e987cc-b78a-41c0-93d3-976e101e4179)

Reply:
Thank you for notice that, i've deleted those code thinking that i didnt need it anymore, my bad, but i putted again in the repository and also we wrote the testing code for all the Decorator design.

I hope that this is good.

![image](https://github.com/lerfast/opp-school-library/assets/111823606/288f1ccf-b6e9-46df-833b-a129a3d5d56b)